### PR TITLE
[ko] Update outdated files in dev-1.21-ko.5 (p3)

### PR DESCRIPTION
**Pull request by jihoon-seo**
Related issue: #28473

> 21. [ ]  M21.  `content/en/docs/concepts/workloads/controllers/daemonset.md`  | 32(+M) 21(-)
> 22. [ ]  M22.  `content/en/docs/concepts/workloads/controllers/garbage-collection.md`  | 1(+XS) 1(-)
> 23. [ ]  M23.  `content/en/docs/concepts/workloads/controllers/job.md`  | 1(+XS) 6(-)
> 24. [ ]  M24.  `content/en/docs/concepts/workloads/pods/disruptions.md`  | 1(+XS) 1(-)
> 25. [ ]  M25.  `content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md`  | 1(+XS) 1(-)
> 26. [ ]  M26.  `content/en/docs/contribute/suggesting-improvements.md`  | 2(+XS) 2(-)
> 27. [ ]  M27.  `content/en/docs/reference/_index.md`  | 1(+XS) 1(-)
> 28. [ ]  M28.  `content/en/docs/reference/access-authn-authz/service-accounts-admin.md`  | 4(+XS) 4(-)
> 29. [ ]  M29.  `content/en/docs/reference/command-line-tools-reference/feature-gates.md`  | 1(+XS) 1(-)

### CODE REVIEW

File: content/ko/docs/concepts/workloads/controllers/daemonset.md [hunk 1]
```diff
@@ -100,18 +114,19 @@ kubectl apply -f https://k8s.io/examples/controllers/daemonset.yaml
...
- * 파드 동작의 불일치: 스케줄 되기 위해서 대기 중인 일반 파드는 `Pending` 상태로 생성된다.
-   그러나 데몬셋 파드는 `Pending` 상태로 생성되지 않는다.
-   이것은 사용자에게 혼란을 준다.
- * [파드 선점](/ko/docs/concepts/configuration/pod-priority-preemption/)은
-   기본 스케줄러에서 처리한다. 선점이 활성화되면 데몬셋 컨트롤러는
-   파드 우선순위와 선점을 고려하지 않고 스케줄 한다.
+* 파드 동작의 불일치: 스케줄 되기 위해서 대기 중인 일반 파드는 `Pending` 상태로 생성된다.
+  그러나 데몬셋 파드는 `Pending` 상태로 생성되지 않는다.
+  이것은 사용자에게 혼란을 준다.
+* [파드 선점](/ko/docs/concepts/configuration/pod-priority-preemption/)은
```
Comment (ClaudiaJKang) [inline content/ko/docs/concepts/workloads/controllers/daemonset.md:120]:
```suggestion
* [파드 선점](/ko/docs/concepts/scheduling-eviction/pod-priority-preemption/)은
```

- 연결되는 링크에 오류가 있어, 변경되어야 할 것 같습니다.

**Review by ClaudiaJKang — COMMENTED** [review]
@jihoon-seo 컨트리뷰션 감사합니다! 리뷰 완료했습니다 :D

**Comment by jihoon-seo** [comment]
@ClaudiaJKang 리뷰 감사합니다!
알려 주신 사항을 반영했습니다.
(링크 주소 변경 1개 라서, squash 까지 하였습니다.)
확인 부탁드립니다 😊

**Comment by ClaudiaJKang** [comment]
@jihoon-seo 감사합니다~ 

/lgtm

**Pull request merged**